### DISCUSSION TITLE
Where does one get a `RETYPE_SECRET`?

### DISCUSSION THREAD

**Discussion by jakerockland** [retypeapp/retype#269] [Q&A]
Following up from https://github.com/ArtBlocks/artblocks-docs/pull/38, I'm not sure where we get a `RETYPE_SECRET` @geoffreymcgill ?

**Comment by geoffreymcgill** [comment]
Oops. Before I noticed your question here, I responded with https://github.com/ArtBlocks/artblocks-docs/pull/38#issuecomment-1032167800.

Decision is yours. If you want to remove the `Powered by Retype` or not. If you want to remove, a key is required. Maybe just sending an email to me hello@retype.com would be easiest.

For others reading this in the future, setting up a `RETYPE_SECRET` secret in your GitHub repo is super simple. Instructions here:

https://retype.com/guides/cli/#retype_secret
